recipes/compute_slurm_config.rb

```diff
@@ -26,8 +26,8 @@
 
 # Mount /opt/slurm over NFS
 # Computemgtd config is under /opt/slurm/etc/pcluster; all compute nodes share a config
-mount '/opt/slurm' do
-  device(lazy { "#{node['cfncluster']['cfn_master_private_ip']}:/opt/slurm" })
+mount (node['cfncluster']['slurm']['install_dir']).to_s do
```

Comment:
Could you remove .to_s here?

Reply:
Yes

Reply:
Actually, sorry, it seems this file is different from what is depicted here.

Reply:
Sorry again, this file is as depicted and I'm pushing the change now